math/isautomorphic.go

```diff
@@ -0,0 +1,34 @@
+// isautomorphic.go
+// description: Checks whether a whole number integer is Automorphic or not. If number < 0 then returns false.
+// details:
+// In mathematics, a number n is said to be a Automorphic number if the square of n ends in the same digits as n itself.
+// ref: (https://en.wikipedia.org/wiki/Automorphic_number)
+// time complexity: O(log10(N))
+// space complexity: O(1)
+// author: [SilverDragonOfR](https://github.com/SilverDragonOfR)
+
+package math
+
+// constraints package: to work with any type of signed/unsigned integer
```

Comment:
You don't need to add this comment - its clear that we are importing a package from this module.